src/TriangleNodesProvisioner.cpp

```diff
@@ -299,16 +336,16 @@ namespace blitzdg {
 
                 index_vector_type ids2(NGauss);
                 for (index_type ind=NGauss-1; ind >= 0; --ind) {
```

Comment:
Just a heads up, if you ever decide to make `index_type` an unsigned type, then this loop's termination criterion would fail. Probably not a big deal, but something to think about.